packages/inferno/src/core/refs.ts

```diff
@@ -29,9 +29,15 @@ export const forwardRef =
         };
       };
 
-export function pushRef(dom: Element | null, value: Function, lifecycle: Function[]) {
+export function pushRef(dom: Element | null, ref, lifecycle: Function[]) {
   lifecycle.push(() => {
-    value(dom);
+    if (ref) {
```

Comment:
is it possible to check if `ref` exists in outer context ? So we can avoid adding function to lifecycle array?

Reply:
Makes sense!